polly/include/polly/Support/PollyDebug.inc

```diff
@@ -0,0 +1,32 @@
+//===-PollyDebug.inc -Provide support for debugging Polly passes-*- C++ -*-===//
```

Comment:
Please use a `.h` extension for this file, it's a normal header file

Reply:
Done